### PR TITLE
Add_columns_unit_time_to_showback_rate

### PR DESCRIPTION
**Pull request by aljesusg**
Added columns in showback rate for steps and units/time
@sergio-ocon

### CODE REVIEW

**Comment by sergio-ocon** [comment]
It adds support for the following use cases for fixed and variable rates:
- Being able to define a rate as $5/MB/day.

It also adds the capability to define steps in rates:
i.e. 2 core minimum, it is not allowed to do 0.78 cores or 1 core, everything will be 0,2,4 etc.

File: db/migrate/20170707162731_add_columns_unit_time_to_showback_rate.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class AddColumnsUnitTimeToShowbackRate < ActiveRecord::Migration[5.0]
+  def change
+    add_column :showback_rates, :fixed_rate_per_unit,    :string
+    add_column :showback_rates, :fixed_rate_per_time,    :string
+    add_column :showback_rates, :variable_rate_per_unit, :string
+    add_column :showback_rates, :variable_rate_per_time, :string
+    add_column :showback_rates, :step_value,             :float
+    add_column :showback_rates, :step_unit,              :string
```
Comment (sergio-ocon) [inline db/migrate/20170707162731_add_columns_unit_time_to_showback_rate.rb]: Add step_per_time_unit step_per_time_value to reflect that the minimum step can be 60 s  or  1h, or 60 m

**Comment by aljesusg** [comment]
@miq-bot remove_label wip

**Comment by sergio-ocon** [comment]
LGTM

@miq-bot assign @Fryguy

**Comment by aljesusg** [comment]
Thanks @Fryguy

**Pull request merged**